shared/chat/conversation/list-area/index.desktop.tsx

```diff
@@ -679,8 +709,9 @@ const OrdinalWaypointInner = (p: OrdinalWaypointProps) => {
   }, [])
 
   // Cache rendered children if the ordinals are the same, else we'll thrash a lot as we scroll up and down
-  const lastVisibleChildrenOrdinalsRef = React.useRef(new Array<T.Chat.Ordinal>())
-  const lastVisibleChildrenRef = React.useRef<React.ReactElement | null>(null)
+  // const lastVisibleChildrenOrdinalsRef = React.useRef(new Array<T.Chat.Ordinal>())
```

Comment:
disable this cache. its not clear the performance win and there's a lot of implicit assumptions here which may be wrong, you have to capture all the dependencies to ensure the render would have ultimately been the same. turn off for now